tst/Carnival/CarnivalTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Tests\Carnival;
+
+use Illuminate\Support\Str;
+use Mockery;
+use Mockery\Exception\InvalidCountException;
+use PHPUnit\Framework\TestCase;
+
+class CarnivalTest extends TestCase {
+    protected function tearDown () : void {
+        parent::tearDown();
+
+        if (class_exists('Mockery')) {
```

Comment:
Will Mockery ever not exist in our case? We're probably safe to remove this condition

Reply:
Okay, I will remove this condition.